components/properties/src/maps.rs

```diff
@@ -61,3 +61,83 @@ where
 {
     get_cp_map(provider, key::SCRIPT_V1)
 }
+
+/// Return a [`CodePointTrie`] for the Grapheme_Cluster_Break Unicode enumerated
+/// property. See [`GraphemeClusterBreak`].
+///
+/// # Example
+///
+/// ```
+/// use icu::properties::{maps, GraphemeClusterBreak};
+/// use icu_provider_uprops::EnumeratedPropertyCodePointTrieProvider;
+///
+/// let root_dir = icu_testdata::paths::uprops_toml_root();
+/// let provider = EnumeratedPropertyCodePointTrieProvider::new(root_dir);
```

Comment:
Nit, here and elsewhere: as of Friday, you can (and should) replace this with `icu_testdata::get_provider()`, which will work after you update your branch and run `cargo make testdata`